docs/src/docs/dates/time-input.mdx

```diff
@@ -36,6 +36,12 @@ To add seconds set `withSeconds` prop:
 
 <Demo data={TimeInputDemos.withSeconds} />
 
+## Format
+
+Switch between `12` and `24` hour time format:
+
+<Demo data={TimeInputDemos.format} />
+
```

Comment:
Change title to `12 hours format`, remove `Switch between 12 and 24 hour time format:` description